views/member/mixins/table.pug

```diff
@@ -0,0 +1,20 @@
+mixin member-table(members, title)
+    h1.hours-table=title
+    table.table
+        thead
+            tr
+                th Name
+                th Email
+                th Major
+                th Meetings
+                th Hours
+                th Not Needed
+        each member in members
+            tr
+                td #{member.first_name} #{member.last_name}
+                td #{member.email}
+                td #{member.major}
+                td #{member.meetings}
+                td #{member.service * 3600000}
+                td #{member.service_not_needed * 3600000}
```

Comment:
This should also be `/` to go from microseconds to hours.

Reply:
Yeah sorry about the mix up. Also is it microseconds or milliseconds?

Reply:
And I agree with your suggestion of renaming the file. hours-table.pug sounds good

Reply:
milliseconds - my bad. That's the right constant